src/test/java/seedu/address/ui/HelpWindowTest.java

```diff
@@ -1,21 +1,29 @@
 package seedu.address.ui;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import org.junit.jupiter.api.Test;
-public class HelpWindowTest {
 
-    @Test
-    public void helpText_containsExpectedContent() {
-        assertTrue(HelpWindow.HELP_TEXT.contains("BA€ 1.0 Help"),
-                "Help text should contain the application name");
-        assertTrue(HelpWindow.HELP_TEXT.contains("Key Features and Commands"),
-                "Help text should mention key features and commands");
-    }
+public class HelpWindowTest {
 
     @Test
     public void helpMessage_containsUserGuideUrl() {
         assertTrue(HelpWindow.HELP_MESSAGE.contains(HelpWindow.USERGUIDE_URL),
                 "Help message should contain the user guide URL");
     }
+
+    @Test
+    public void userGuideUrl_isCorrect() {
+        assertEquals("https://ay2425s1-cs2103t-w14-2.github.io/tp/UserGuide.html",
```

Comment:
Unsure if there are better ways of specifying or storing the expected, actual User Guide URL. Perhaps dynamically generating it on compile/build will be technically better, but objectively will be much more complicated to implement and maintain.

But this is a very useful safety net check for now. Good catch.